### PR TITLE
Update dataplane refs

### PR DESCRIPTION
**Pull request by fao89**

### CODE REVIEW

File: edpm_service_overview.md [hunk 1]
```diff
@@ -8,28 +8,28 @@ which needs to be hosted on a RHEL-based External Dataplane Management
...
 for a new service should contain an `OpenStackDataPlaneService`
-definition ([examples](https://github.com/openstack-k8s-operators/dataplane-operator/tree/main/config/services))
+definition ([examples](https://github.com/openstack-k8s-operators/openstack-operator/tree/main/config/services))
 and a sample of how to use the service in an `OpenStackDataPlaneNodeSet`
-([examples](https://github.com/openstack-k8s-operators/dataplane-operator/tree/main/config/samples)).
+([examples](https://github.com/openstack-k8s-operators/openstack-operator/tree/main/config/samples)).
 
 See the
-[Data Plane Operator documentation](https://openstack-k8s-operators.github.io/dataplane-operator)
+[Data Plane Operator documentation](https://openstack-k8s-operators.github.io/openstack-operator)
```
Comment (fultonj) [inline edpm_service_overview.md:26]: Should this instead refer to the "OpenStack Operator documentation" ?

**Review by fultonj — COMMENTED** [review]
I'm +2 on these changes, though there's one proposed change I have a question about above (since it's still using the term "data plane operator").

File: ceph.md [hunk 2]
```diff
@@ -510,4 +510,4 @@ CR can be modified to include Ceph configuration by adding
 examples are below.
 
 - `OpenStackControlPlane`: [core_v1beta1_openstackcontrolplane_network_isolation_ceph.yaml](https://github.com/openstack-k8s-operators/openstack-operator/blob/main/config/samples/core_v1beta1_openstackcontrolplane_network_isolation_ceph.yaml)
-- `OpenStackDataPlaneNodeSet`: [dataplane_v1beta1_openstackdataplanenodeset_ceph.yaml](https://github.com/openstack-k8s-operators/dataplane-operator/blob/main/config/samples/dataplane_v1beta1_openstackdataplanenodeset_ceph.yaml)
+- `OpenStackDataPlaneNodeSet`: [dataplane_v1beta1_openstackdataplanenodeset_ceph.yaml](https://github.com/openstack-k8s-operators/openstack-operator/blob/main/config/samples/dataplane_v1beta1_openstackdataplanenodeset_ceph.yaml)
```
[the remark this answers is not in the document]
Comment (gibizer) [reply ceph.md:513]: I only see https://github.com/openstack-k8s-operators/openstack-operator/blob/main/config/samples/core_v1beta1_openstackcontrolplane_network_isolation_ceph.yaml but I'm not sure it is equivalent.

File: hci.md [hunk 3]
```diff
@@ -131,7 +131,7 @@ communicate at the desired MTU with a command like `ping -M do -s 8972
 ### Create the OpenStackDataPlaneNodeSet
 
 Create the CR from your file based on the example
-[dataplane_v1beta1_openstackdataplanenodeset_ceph_hci.yaml](https://github.com/openstack-k8s-operators/dataplane-operator/blob/main/config/samples/dataplane_v1beta1_openstackdataplanenodeset_ceph_hci.yaml)
+[dataplane_v1beta1_openstackdataplanenodeset_ceph_hci.yaml](https://github.com/openstack-k8s-operators/openstack-operator/blob/main/config/samples/dataplane_v1beta1_openstackdataplanenodeset_ceph_hci.yaml)
```
Comment (gibizer) [inline hci.md:134]: 404

File: hci.md [hunk 4]
```diff
@@ -149,7 +149,7 @@ of services of an `OpenStackDataPlaneNodeSet` for a
 deployment.
 
 The example
-[dataplane_v1beta1_openstackdataplanedeployment_pre_ceph_hci.yaml](https://github.com/openstack-k8s-operators/dataplane-operator/blob/main/config/samples/dataplane_v1beta1_openstackdataplanedeployment_pre_ceph_hci.yaml)
+[dataplane_v1beta1_openstackdataplanedeployment_pre_ceph_hci.yaml](https://github.com/openstack-k8s-operators/openstack-operator/blob/main/config/samples/dataplane_v1beta1_openstackdataplanedeployment_pre_ceph_hci.yaml)
```
Comment (gibizer) [inline hci.md:152]: 404

File: hci.md [hunk 5]
```diff
@@ -161,7 +161,7 @@ oc create -f openstackdataplanedeployment_pre_ceph_hci.yaml
 #### Pre-Ceph Service Details
 
 The example
-[dataplane_v1beta1_openstackdataplanedeployment_pre_ceph_hci.yaml](https://github.com/openstack-k8s-operators/dataplane-operator/blob/main/config/samples/dataplane_v1beta1_openstackdataplanedeployment_pre_ceph_hci.yaml)
+[dataplane_v1beta1_openstackdataplanedeployment_pre_ceph_hci.yaml](https://github.com/openstack-k8s-operators/openstack-operator/blob/main/config/samples/dataplane_v1beta1_openstackdataplanedeployment_pre_ceph_hci.yaml)
```
Comment (gibizer) [inline hci.md:164]: 404

File: hci.md [hunk 6]
```diff
@@ -188,7 +188,7 @@ list above. For more informatoin, see the `ceph-hci-pre` role in the
 [edpm-ansible role documentation](https://openstack-k8s-operators.github.io/edpm-ansible/roles.html).
 
 As seen in the example
-[dataplane_v1beta1_openstackdataplanedeployment_pre_ceph_hci.yaml](https://github.com/openstack-k8s-operators/dataplane-operator/blob/main/config/samples/dataplane_v1beta1_openstackdataplanedeployment_pre_ceph_hci.yaml),
+[dataplane_v1beta1_openstackdataplanedeployment_pre_ceph_hci.yaml](https://github.com/openstack-k8s-operators/openstack-operator/blob/main/config/samples/dataplane_v1beta1_openstackdataplanedeployment_pre_ceph_hci.yaml),
```
Comment (gibizer) [inline hci.md:191]: 404

File: edpm_service_overview.md [hunk 7]
```diff
@@ -8,28 +8,28 @@ which needs to be hosted on a RHEL-based External Dataplane Management
 
 Though the service may not run directly on OpenShift workers, the
 service definition must be made through the Custom Resources of the
-[Data Plane Operator](https://openstack-k8s-operators.github.io/dataplane-operator)
+[Data Plane Operator](https://openstack-k8s-operators.github.io/openstack-operator)
```
[the remark this answers is not in the document]
Comment (gibizer) [reply edpm_service_overview.md:11]: maybe you want to link here for the CRDs https://openstack-k8s-operators.github.io/openstack-operator/dataplane/#custom-resources

File: nfv_ovs_dpdk.md [hunk 8]
```diff
@@ -116,14 +116,14 @@ This example also assumes that the EDPM nodes:
...
-[dataplane_v1beta1_openstackdataplanenodeset.yaml](https://github.com/openstack-k8s-operators/dataplane-operator/blob/main/config/samples/dataplane_v1beta1_openstackdataplanenodeset.yaml)
+[dataplane_v1beta1_openstackdataplanenodeset.yaml](https://github.com/openstack-k8s-operators/openstack-operator/blob/main/config/samples/dataplane_v1beta1_openstackdataplanenodeset.yaml)
 for an example to modify as described in this document.
 
 Do not yet create the CR in OpenShift as the edits described in the
 next sections are required.
 
 The example
-[dataplane_v1beta1_openstackdataplanenodeset_ovs_dpdk.yaml]https://github.com/openstack-k8s-operators/dataplane-operator/tree/main/examples/ovs_dpdk)
+[dataplane_v1beta1_openstackdataplanenodeset_ovs_dpdk.yaml]https://github.com/openstack-k8s-operators/openstack-operator/tree/main/config/samples/dataplane/ovs_dpdk)
```
Comment (gibizer) [inline nfv_ovs_dpdk.md:126]: that this link missing the '(' to make it valid?

File: release_and_branching.md [hunk 9]
```diff
@@ -38,11 +38,6 @@ the following extra steps:
 
 * It contains a dependencies.yaml file which drives OLM dependencies to install all other OpenStack service operators
 * When the bundle is generated it runs a script to *merge* all the ENV variables from each service operators controller-manager. These ENV variables contain important webhook defaults that are needed within the OpenStack operator.
-* A csv-merger is used to merge in the CSV from the dataplane-operator along with all the ENV variables harvested from the above.
```
Comment (gibizer) [inline release_and_branching.md:41]: I think this part "along with all the ENV variables harvested from the above." is still valid. The csv-merger merges in the collected ENV variable into the CSV.

**Comment by fao89** [comment]
Thank you @fultonj and @gibizer

**Review by gibizer — APPROVED** [review]
thanks. looks good to me.

**Pull request merged**